### DISCUSSION TITLE
Does `git_files` accept actions?

### DISCUSSION THREAD

**Discussion by dpetka2001** [ibhagwan/fzf-lua#1254] [Q&A] [answered]
I'm trying the following setup
```lua
{
  "ibhagwan/fzf-lua",
  opts = function(_, opts)
      local actions = require("fzf-lua.actions")
      opts.files = {
        actions = {
          ["alt-i"] = { actions.toggle_ignore },
          ["alt-h"] = { actions.toggle_hidden },
        },
      }
      opts.git = {
        files = {
          actions = {
            ["alt-i"] = { actions.toggle_ignore },
            ["alt-h"] = { actions.toggle_hidden },
          },
        },
      }
  end,
}
```
While `FzfLua files` correctly toggles hidden files on/off I cannot achieve the same behavior with `FzfLua git_files`. Am I doing something wrong?

**Comment by dpetka2001** [comment]
Ok, I'm guessing that's not possible with `git ls-files`, right?

**Reply by ibhagwan** [reply, marked as answer]
The actions you linked toggle `--hidden` and `--no-ignore` respectively on the shell command, these aren’t valid slags for `git ls-files` as you found out. You can change the flag the action toggles but I’m not sure if there’s an option to show hidden files.
